resources/definitions/Mark2_for_Ultimaker2.def.json

```diff
@@ -0,0 +1,241 @@
+        },
+        "machine_end_gcode" : {
+            "default_value": "",
+            "value": "\"\"  if machine_gcode_flavor == \"UltiGCode\" else \"G90 ;absolute positioning\\nM104 S0 T0 ;extruder heater off\\nM104 S0 T1\\nM140 S0 ;turn off bed\\nT0 ; move to the first head\\nM107 ;fan off\""
+        },
+        "machine_extruder_count": {
+            "default_value": 2
+        },
+        "acceleration_enabled":
+        {
+            "default_value": true
+        },
+        "acceleration_print":
+        {
+            "default_value": 2000,
+            "value": "2000"
+        },
+        "acceleration_travel":
+        {
+            "default_value": 3000,
+            "value": "acceleration_print if magic_spiralize else 3000"
+        },
+        "acceleration_layer_0": { "value": "acceleration_topbottom" },
+        "acceleration_prime_tower": { "value": "math.ceil(acceleration_print * 2000 / 4000)" },
+        "acceleration_support": { "value": "math.ceil(acceleration_print * 2000 / 4000)" },
+        "acceleration_support_interface": { "value": "acceleration_topbottom" },
+        "acceleration_topbottom": { "value": "math.ceil(acceleration_print * 500 / 4000)" },
+        "acceleration_wall": { "value": "math.ceil(acceleration_print * 1000 / 4000)" },
+        "acceleration_wall_0": { "value": "math.ceil(acceleration_wall * 500 / 1000)" },
+        "jerk_enabled":
+        {
+            "default_value": true
+        },
+        "jerk_print":
+        {
+            "default_value": 12
+        },
+        "jerk_travel":
+        {
+            "default_value": 20,
+            "value": "jerk_print if magic_spiralize else 20"
+        },
+        "jerk_layer_0": { "value": "jerk_topbottom" },
+        "jerk_prime_tower": { "value": "10 if jerk_print < 16 else math.ceil(jerk_print * 15 / 25)" },
+        "jerk_support": { "value": "10 if jerk_print < 16 else math.ceil(jerk_print * 15 / 25)" },
+        "jerk_support_interface": { "value": "jerk_topbottom" },
+        "jerk_topbottom": { "value": "10 if jerk_print < 25 else math.ceil(jerk_print * 10 / 25)" },
+        "jerk_wall": { "value": "10 if jerk_print < 16 else math.ceil(jerk_print * 15 / 25)" },
+        "jerk_wall_0": { "value": "10 if jerk_wall < 16 else math.ceil(jerk_wall * 6 / 10)" },
+        "jerk_travel_layer_0": { "value": "math.ceil(jerk_layer_0 * jerk_travel / jerk_print)" },
+        "extruder_prime_pos_abs": { "default_value": false },
+        "machine_extruder_start_pos_abs": { "default_value": false },
```

Comment:
Those fields should not be in the machine definition files. CI is failing due to this problem. Please remove the following fields from this file:
```
machine_extruder_start_pos_abs
machine_extruder_start_pos_x
machine_extruder_start_pos_y
machine_extruder_end_pos_abs
machine_extruder_end_pos_x
machine_extruder_end_pos_y
```

Reply:
Done. It seems like checks were successful this time 😃